src/drivers/uart/src/i8250.rs

```diff
@@ -1,47 +1,19 @@
-//use core::ops;
 use model::*;
-//use register::mmio::{ReadOnly, ReadWrite};
-//use register::{register_bitfields, Field};
-
-/*
-#[repr(C)]
-pub struct RegisterBlock {
-    d: ReadWrite<u8, D::Register>,
-    ie: ReadWrite<u8, IE::Register>,
-    fc: ReadWrite<u8, FC::Register>,
-    lc: ReadWrite<u8, LC::Register>,
-    mc: ReadWrite<u8, MC::Register>,
-    ls: ReadOnly<u8, LS::Register>,
-}
-*/
+
 pub struct I8250<'a> {
     base: usize,
-    //baud: u32,
+    _baud: u32,
     d: &'a mut dyn Driver,
 }
 
-// it is possible that trying to make this work is a fool's errand but
-// ... would be nice if the deref used the Driver in the 8250 ... dream on.
-// impl ops::Deref for I8250 {
-//     type Target = RegisterBlock;
-
-//     fn deref(&self) -> &Self::Target {
-//         unsafe { &*self.ptr() }
-//     }
-// }
-
 impl<'a> I8250<'a> {
-    pub fn new(base: u16, _baudrate: u32, d: &'a mut dyn Driver) -> I8250<'a> {
-        I8250 { base: base as usize, d: d }
+    // why is base a usize? for mmio 8250.
```

Comment:
What is the question? https://doc.rust-lang.org/std/primitive.usize.html "The size of this primitive is how many bytes it takes to reference any location in memory. For example, on a 32 bit target, this is 4 bytes and on a 64 bit target, this is 8 bytes."